containers/python-bootstrap-container/.repo-metadata.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "apiName",
+    "name": "apiPackage",
     "name_pretty": "apiPrettyName",
     "api_description": "",
     "product_documentation": "apiProductDocumentation",
-    "client_documentation": "https://cloud.google.com/python/docs/reference/apiName/latest",
+    "client_documentation": "https://cloud.google.com/python/docs/reference/apiPackage/latest",
```

Comment:
This is superseded by #11959 which overwrites `client_documentation`, right?

Reply:
There are 2 different issues being addressed in this PR and #11959, but yes https://github.com/googleapis/google-cloud-python/pull/11959 also updates this line.


#11958 addresses https://github.com/googleapis/google-cloud-python/issues/11957
#11959 addresses https://github.com/googleapis/google-cloud-python/issues/11193